### PR TITLE
READY FOR MERGING: Sagittarius engine

### PR DESCRIPTION
**Pull request by astronautthis**
Staging link: https://makerdao-1.gitbook.io/endgame-docs-staging/tokenomics/sagittarius-engine

### CODE REVIEW

File: tokenomics/sagittarius-engine.md [hunk 1]
```diff
@@ -1,32 +1,52 @@
...
+The Sagittarius Engine is a module that incentivizes MKR holders to lock their MKR tokens up for long periods of time. The locked-up MKR is required to participate in Maker Governance via delegation. 
 
-{% hint style="warning" %} This documentation describes planned functionality and processes that MakerDAO has not yet implemented. Be aware that parts may be inaccurate or out of date. {% endhint %}
+The incentives for locking up MKR in the Sagittarius Engine are:
+- Token farming
+- Dai generation through vaults with MKR as collateral
 
-## Usage
-Users may deposit MKR into the Sagittarius Engine. When depositing, they automatically delegate the deposited MKR through the Protocol Delegation System (PDS). Depositors must specify a target delegate smart contract through the PDS when making their deposit.
+Withdrawing MKR from the Sagittarius Engine burns part of the deposited MKR.
```
Comment (LongForWisdom) [inline tokenomics/sagittarius-engine.md:10]: My (old, possibly out-of-date) understanding was that this was only if you didn't wait the required lock-up period before exiting?

File: tokenomics/sagittarius-engine.md [hunk 2]
```diff
@@ -1,32 +1,52 @@
...
 
-## Farming
-When users make their deposit, they can choose one of several ERC-20 tokens to farm. This functionality will initially be disabled with no available tokens to farm. 
+{% hint style="warning" %} This documentation describes planned functionality and processes that MakerDAO has not yet implemented. Be aware that parts may be inaccurate or out of date. {% endhint %}
 
-Maker Governance can later add farmable SubDAO governance tokens, making the first version of the Sagittarius Engine compatible with the launch of SubDAOs.
+## Entry and Exit
+When users deposit MKR into the Sagittarius Engine, they must delegate the deposited MKR through the [Easy Governance Frontend (EGF)](../maker-core/easy-governance-frontend.md). Depositors must specify a target delegate smart contract when making their deposit.
```
Comment (LongForWisdom) [inline tokenomics/sagittarius-engine.md:15]:
```suggestion
When users deposit MKR into the Sagittarius Engine, they must delegate the deposited MKR through the [Easy Governance Frontend (EGF)](../maker-core/easy-governance-frontend.md). Depositors must specify a delegate's strategy smart contract when making their deposit.
```

Given delegates will having multiple contracts attached to strategies. (afaik)

File: tokenomics/sagittarius-engine.md [hunk 3]
```diff
@@ -1,32 +1,52 @@
...
+
+#### Price Oracle
+The oracle for MKR price uses [Elixir](elixir.md) as the primary data source. 
 
-The oracle for MKR prices used to manage these MKR vaults has constrained upwards movement. That is, Maker Governance can choose to limit the speed at which the reported price of MKR increases, regardless of how fast the price is actually increasing. This constraint does not apply to downward price movement. 
+A "sticky price" feature limits upward movement of the price within a fixed time period. That is, Maker Governance can choose to limit the speed at which the reported price of MKR increases, regardless of how fast the Elixir price is increasing. This constraint does not apply to downward price movement. 
 
-When an MKR vault is liquidated, the MKR is undelegated and put up for auction. Token farming also stops when a vault is liquidated.
+#### [Liquidation Ratio](https://manual.makerdao.com/parameter-index/vault-risk/param-liquidation-ratio)
+The Liquidation Ratio is set to 200\%. If a vault falls below this collateralization ratio, it is liquidated completely using Elixir order books.
```
Comment (LongForWisdom) [inline tokenomics/sagittarius-engine.md:34]: Backslash for percentage?

File: tokenomics/sagittarius-engine.md [hunk 4]
```diff
@@ -1,32 +1,52 @@
...
+
+Sagittarius Engine MKR Vaults also have an additional parameter called the Soft Liquidation Ratio, which is set to 300\%. If a vault's collateralization ratio stays below this for a continuous 7-day period, it is liquidated.
```
Comment (LongForWisdom) [inline tokenomics/sagittarius-engine.md:36]: Also important that users can't open a vault at less than 300% CR.

**Review by LongForWisdom — CHANGES_REQUESTED** [review]
Minor changes needed (see comments) but overall really good. Thanks Astro!

File: tokenomics/sagittarius-engine.md (hunk 1 above)
Comment (astronautthis) [reply tokenomics/sagittarius-engine.md:10]: The latest post doesn't have a lockup period. It reads as if 15% of the deposited MKR is burned upon exiting, regardless of when you do it.

File: tokenomics/sagittarius-engine.md (hunk 3 above)
Comment (astronautthis) [reply tokenomics/sagittarius-engine.md:34]: Latex hangover. Apparently markdown doesn't need the backslash!

File: tokenomics/sagittarius-engine.md (hunk 1 above)
Comment (astronautthis) [reply tokenomics/sagittarius-engine.md:10]: Can be merged if you're happy with this point about the lockup period.

**Pull request merged**